client/js/bundles/procedure/administrationStatementSegmentsList.js

```diff
@@ -26,8 +26,7 @@ const components = {
 const stores = {
   ProcedureMapSettings: procedureMapSettings,
   SegmentSlidebar,
-  SplitStatement: SplitStatementStore,
-  Voter
```

Comment:
We still need this store for the old views.

Reply:
Done in [7d0f569](https://github.com/demos-europe/demosplan-core/pull/3996/commits/7d0f56911d2eed1bd771fb21bd4fb1e8418b590b)